yarn.lock

```diff
@@ -23336,6 +23348,13 @@ react-addons-create-fragment@^15.6.2:
     loose-envify "^1.3.1"
     object-assign "^4.1.0"
 
+react-app-rewired@^2.1.6:
```

Comment:
Why do we have `react-app-rewired` in our root `yarn.lock`? I think this only need in our created app using scaffolder?